frontend/src/components/Courses/CourseContent.jsx

```
@@ -59,54 +55,141 @@ const PDFContent = ({ pdfUrl }) => (
+						onClick={handlePrevious}
+						disabled={currentIndex === 0}
+					>
+						<FaArrowLeft />
+						Prev
+					</button>
+
+					<button
+						className="btn btn-sm flex items-center gap-2 bg-blue-200 disabled:opacity-50 disabled:cursor-not-allowed"
+						onClick={handleNext}
+						disabled={currentIndex === contents.length - 1}
+					>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)